src/raftstore/store/fsm/store.rs

```diff
@@ -343,16 +344,18 @@ impl<T: Transport, C> PollContext<T, C> {
 
         if !need_gc {
             info!(
-                "[region {}] raft message {:?} is stale, current {:?}, ignore it",
-                region_id, msg_type, cur_epoch
+                "raft message {:?} is stale, ignore it", msg_type;
```

Comment:
It is wired to use two type of log format.

Reply:
why not making msg_type a field?